documentation/library_guide/kernel_templates/esimd_main.rst

```diff
@@ -0,0 +1,23 @@
+ESIMD-based Kernel Templates
+############################
+
+ESIMD Kernel Templates are based on `Explicit SIMD SYCL (ESIMD) <https://www.intel.com/content/www/us/en/docs/dpcpp-cpp-compiler/developer-guide-reference/2024-0/explicit-simd-sycl-extension.html>`_.
```

Comment:
My suggestions (note that length is spelled incorrectly in vector_lenght):

The ESIMD kernel templates are based on `Explicit SIMD SYCL (ESIMD) <https://www.intel.com/content/www/us/en/docs/dpcpp-cpp-compiler/developer-guide-reference/2024-0/explicit-simd-sycl-extension.html>`_. This technology only supports Intel GPU devices.

NOTE:
The ``kernel_param::data_per_workitem`` parameter has a special meaning in ESIMD-based kernel templates. Usually, each work item processes ``data_per_workitem`` sequentially. However, work items in ESIMD-based kernel templates perform vectorization, so the sequential work is ``data_per_workitem / vector_lenght``, where ``vector_lenght`` is an implementation-defined vectorization factor.

Reply:
I have fixed the length spelling. It seems the rest is the same besides "work-item"/"work item" which we agreed to leave as is.

Reply:
LGTM